manifests/components/blackbox-exporter.jsonnet

```diff
@@ -0,0 +1,107 @@
+// vim set syntax=jsonnet
+// Imports
+local kube = import "../vendor/github.com/bitnami-labs/kube-libsonnet/kube.libsonnet";
+local kubecfg = import "kubecfg.libsonnet";
+
+// Cluster-specific configuration
+local BLACKBOX_EXPORTER_IMAGE = "bitnami/blackbox-exporter:0.18.0-debian-10-r83";
+
+{
+  local this = self,
+
+  be_config:: {
```

Comment:
Could you explain this configuration?

Reply:
This is just the default config from https://github.com/prometheus/blackbox_exporter/blob/master/blackbox.yml.